### ISSUE TITLE
Clipboard function broken 

### ISSUE THREAD

**Issue by desmonk** [openboard-team/openboard#677]
To reproduce the issue, please try these steps: 

App version: 1.4.5

1. Enable Clipboard with history retention time to one minute limit for this quick test.
2. Copy some text and paste to confirm functionality.
3. Wait for the timer to clear the history.
4. Goto the clipboard history panel, the window where the copied texts appear, will show it as empty.
5. In reality though you will still be able to paste last copied text of clipboard by using press and hold finger gesture.

Only way to clear history is to click on that history deletion button to the bottom right corner. Earlier versions are immune to this flaw.

**Comment by MajeurAndroid** [comment]
Yes, this was something to figure out during development. The choice that has been taken was to avoid OpenBoard clearing system history without user knowing it.
Not doing so would, for example, lead into user copying something and losing it one minute later even if OpenBoard has not been opened. This would appear as a weird behaviour to user.
Still, if you figure out a better solution, feel free to share it.

**Comment by desmonk** [comment]
Clearing entire clipboard history is really a well thoughout move and now with a timer to automate this completely. Rather, this is more like a well made security feature! Not only unaware users won't leave clipboard accessible to other apps but also now know that this new timer got their back!

Another point here is that this new behavior gives a false impression of history being cleared and that can lead to clipboard history available indefinitely as when we look at the history itself, it will show it as empty and unless we use traditional long press method of paste, users will think that the timer has cleared it by now until we manually clear using the separate button.

The timer is a great feature and should be endorsed as a security feature instead which can now take care of clipboard history automatically! Users now have a choice. They can keep the history and use manual way of clearing or with the timer but in both of these cases, the history dashboard should show the correct clipboard history until cleared by any means.

**Comment by rudolfmu** [comment]
Please let the value 0 disable the clearing of the clipboard. Like it was before 1.4.5.
The timer is useless for security. To see this install Clipstack and you will notice it catches everything no matter how you set the timer.
https://www.f-droid.org/packages/com.catchingnow.tinyclipboardmanager/

**Comment by desmonk** [comment]
Some thoughts after long term usage. Problem is the clear button not clearing the clipboard. I understand what you have conveyed before but this is what clear function should do when user chooses to manually clear everything and clear the clipboard completely.

The auto clear timer function too should follow the route and clear after user selected time so no other apps can misuse any clipboard data and this surely will increase privacy and security.

If you still plan on keeping the last copied clipboard data as it is then how about adding an option for that in addition to above points. This means that user can now choose easily from the three ways to handle clipboard.

To summarize,
1. Manual Mode --> The clear button leans everything including last copied data
2. Timer Based --> Same as manual but with a timer leaving nothing behind
3. Keep Last Copy --> This will keep last copied data while still removing the rest

Hope you @MajeurAndroid like the idea!
